scripts/commands/addlights.lua

```diff
@@ -0,0 +1,66 @@
+---------------------------------------------------------------------------------------------------
+-- func: addlights <light type> <amount> <target player>
+-- desc: Adds the amount of specified light to the player
+---------------------------------------------------------------------------------------------------
+-- lights: 1: pearl, 2: azure, 3: ruby, 4: amber, 5: gold, 6: silver, 7: ebon
+---------------------------------------------------------------------------------------------------
+require("scripts/globals/abyssea")
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "sis"
+}
+
+local lightType =
+{
+    pearl  = 1,
+    azure  = 2,
+    ruby   = 3,
+    amber  = 4,
+    gold   = 5,
+    silver = 6,
+    ebon   = 7,
+}
+
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!addlights <light type> <amount> {player}")
+end
+
+function onTrigger(player,light,amount,target)
+    -- validate target
+    local targ
+    if (target == nil) then
```

Comment:
```lua
    if target then
        targ = GetPlayerByName(target)
        if not targ then
            error(player, string.format("Player named '%s' not found!", target))
            return
        end
    else
        targ = player
    end
```
would be preferred formatting.

other similar examples using `== nil` and extra parentheses exist in this pr as well. Does not affect functionality, but I thought you'd like to know to save yourself extra typing in the future.